docs/user-guide/layer-settings.md

```diff
@@ -134,6 +139,110 @@ The editor is easy to approach thanks also to the following functions:
 !!!warning
     The *autocomplete* and the *color picker* functions are available only in the CSS editor.
 
+### Visual Editor Style
+
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) also allows to edit the layers style using the *Visual editor*: clicking on the <img src="../img/button/visual_editor_style_button.jpg" class="ms-docbutton"/> button a page opens allows the user to customize the style through the symbolizers, which can be: *Mark*, *Icon*, *Line*, *Fill* and *Text*. 
+
+<img src="../img/layer-settings/visual_editor_style.jpg" class="ms-docimage"  style="max-width:500px;">
+
+Once a symbolizer has been added and customized, you can:
+
+<img src="../img/layer-settings/style_options.jpg" class="ms-docimage"  style="max-width:500px;">
+
+* **Filter** the symbolizer, as explained [here](filtering-layers.md#attribute-filter), by clicking the <img src="../img/button/filter_white_button.jpg" class="ms-docbutton"/> button.
+
+* Add a **Scale denominator filter** (`max` and `min` scale) within which the level will be displayed on the map by clicking the <img src="../img/button/scale_denominator_button.jpg" class="ms-docbutton"/> button.
+
+* **Remove** the symbolizer by clicking the <img src="../img/button/delete_white_button.jpg" class="ms-docbutton"/> button.
+
+#### Mark
+
+The mark type allows you to add a mark to the layer: clicking on the <img src="../img/button/add_mark_button.jpg" class="ms-docbutton"/> button a mark panel appears: 
+
+<img src="../img/layer-settings/mark_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The mark can have different `Shape`, `Color`, `Stroke` with different `Color` and `Width` and customizable `Radius` and `Rotation`. Take a look at the following example.
+
+<img src="../img/layer-settings/mark_style_ex.jpg" class="ms-docimage">
+
+#### Icon 
+
+With the icon panel, which opens by clicking on <img src="../img/button/add_icon_button.jpg" class="ms-docbutton"/> button, the editor is allowed to add an image as an icon (via its *URL*) and customize the icon `Opacity`, `Size` and `Rotation` angle:
+
+<img src="../img/layer-settings/icon_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+#### Line
+
+The line types allows you to customize the lines of the layer: clicking on the <img src="../img/button/add_line_button.jpg" class="ms-docbutton"/> button a line panel appears: 
```

Comment:
The Line rule is used to style linear features of the layer: clicking on the <img src="../img/button/add_line_button.jpg" class="ms-docbutton"/> button a panel allows the user to edit the corresponding properties.